pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/service/mail/MailTemplateService.java

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.ddp.service.mail;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import one.util.streamex.StreamEx;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.ddp.db.dao.MailTemplateDao;
+import org.broadinstitute.ddp.db.dao.MailTemplateRepeatableElementDao;
+import org.broadinstitute.ddp.db.dto.MailTemplateDto;
+import org.broadinstitute.ddp.db.dto.MailTemplateRepeatableElementDto;
+import org.broadinstitute.ddp.exception.DDPException;
+import org.broadinstitute.ddp.model.mail.MailTemplate;
+import org.broadinstitute.ddp.model.mail.MailTemplateRepeatableElement;
+import org.jdbi.v3.core.Handle;
+
+import java.util.HashMap;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class MailTemplateService {
+    public static MailTemplate getTemplate(final Long id) {
```

Comment:
This looks like it should be `long`, not `Long`- a `null` here seems like it should be treated as an error before this point.